examples/tv-app/android/java/ContentAppCommandDelegate.cpp

```diff
@@ -321,11 +321,11 @@ GetSetupPINResponseType ContentAppCommandDelegate::FormatGetSetupPINResponse(Jso
         std::to_string(to_underlying(app::Clusters::AccountLogin::Commands::GetSetupPINResponse::Fields::kSetupPIN));
     if (!value[setupPINFieldId].empty())
     {
-        getSetupPINresponse.setupPIN.SetNonNull(CharSpan::fromCharString(value[setupPINFieldId].asCString()));
+        getSetupPINresponse.setupPIN = CharSpan::fromCharString(value[setupPINFieldId].asCString());
```

Comment:
Fwiw, pre-existing but I am pretty sure this is a use-after-free in general....

Reply:
created #30155 for this